### PR TITLE
DevTools Tips 12 > Animations tab refresh + promo post

### LINKED ISSUE #3205: [devtools-tips] 12 - Animation tab, device emulation
Details: b/228175919
Deadline: Aug 10
Notes: Do not merge the PR when ready, tag @jecfish to merge in time with video publish
Also refresh relevant docs, and include the video

### PR DESCRIPTION
**Pull request by sofiayem**
Fixes #3205 

Refreshed a 2016 doc about the Animations tab prior to the release of the corresponding DTT 12 video.
Drafted a promo post for the video.
Also, documented the missing Elements > Styles > Easing editor.

Publish on Aug 11.

### CODE REVIEW

**Comment by jecfish** [comment]
@jh3y can you help to review the documentation for Animations tab to see if we've covered everything correctly?

Animations tab: https://deploy-preview-3414--developer-chrome-com.netlify.app/docs/devtools/css/animations/
easing: https://deploy-preview-3414--developer-chrome-com.netlify.app/docs/devtools/css/reference/#edit-easing

**Comment by sofiayem** [comment]
Added the promo post. Ready for review.
Publish tomorrow in sync with the video release.
PTAL @jh3y @jecfish @rachelandrew

File: site/en/blog/devtools-tips-12/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+hero: 'image/NJdAV9UgKuN8AhoaPBquL7giZQo1/ZcQHNwVHsQ3dIVNWbUAH.png'
+alt: >
+  DevTools Tips hero logo
+tags:
+  - css
+  - devtools
+  - devtools-tips
+---
+
+The Chrome DevTools **Animations** tab is a powerful tool that helps you slow down, replay, nail CSS animation timings, and see the results right away.
```
Comment (rachelandrew) [inline site/en/blog/devtools-tips-12/index.md:19]: I'm wondering about how "nail CSS animation timings" might be understood. It reads a bit strangely to me.

**Review by rachelandrew — APPROVED** [review]
LGTM, not a blocker but I'm not totally sold by the opening line (see comment).

File: site/en/blog/devtools-tips-12/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: >
+  DevTools Tips: How to inspect and modify CSS animations
```
Comment (jecfish) [inline site/en/blog/devtools-tips-12/index.md:3]:
```suggestion
  DevTools Tips: How to inspect and modify animations
```

**Review by jecfish — APPROVED** [review]

**Pull request merged**